From 1da35513a7bca12209874ea7208b60042ea1de41 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Tue, 11 May 2021 16:24:33 +0200
Subject: [PATCH 32/34] riscv: defconfigs: starlight: Enable support for NFS
 root

Enable support for obtaining network information from DHCP, and mount
the root file system over NFS.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 arch/riscv/configs/starlight_defconfig | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/riscv/configs/starlight_defconfig b/arch/riscv/configs/starlight_defconfig
index b33a6e36fbdd..0b471baad01c 100644
--- a/arch/riscv/configs/starlight_defconfig
+++ b/arch/riscv/configs/starlight_defconfig
@@ -52,6 +52,8 @@ CONFIG_UNIX=y
 CONFIG_INET=y
 CONFIG_IP_ADVANCED_ROUTER=y
 CONFIG_IP_MULTIPLE_TABLES=y
+CONFIG_IP_PNP=y
+CONFIG_IP_PNP_DHCP=y
 # CONFIG_INET_DIAG is not set
 # CONFIG_IPV6_SIT is not set
 CONFIG_IPV6_MULTIPLE_TABLES=y
@@ -167,7 +169,8 @@ CONFIG_PROC_KCORE=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
 # CONFIG_MISC_FILESYSTEMS is not set
-# CONFIG_NETWORK_FILESYSTEMS is not set
+CONFIG_NFS_FS=y
+CONFIG_ROOT_NFS=y
 CONFIG_NLS_DEFAULT="utf8"
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_15=y
-- 
2.30.0

